am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -30,23 +27,19 @@ private DefaultRoleSetupDataFactory() {
         return ImmutableMap.of(attribute, pair);
     }
 
-    public static DefaultPermissionGrant createDefaultPermissionGrant(Set<Permission> permissions) {
-        return createDefaultPermissionGrant(JsonPointer.valueOf(""), permissions, PUBLIC);
+    public static DefaultPermissionGrant createDefaultPermissionGrant(ResourceDefinition resourceDefinition, Set<Permission> permissions) {
+        return createDefaultPermissionGrant(JsonPointer.valueOf(""), permissions, resourceDefinition, PUBLIC);
     }
 
-    public static DefaultPermissionGrant createDefaultPermissionGrant(JsonPointer attribute, Set<Permission> permissions) {
-        return createDefaultPermissionGrant(attribute, permissions, PUBLIC);
+    public static DefaultPermissionGrant createDefaultPermissionGrant(JsonPointer attribute, ResourceDefinition resourceDefinition, Set<Permission> permissions) {
```

Comment:
Please keep consistent order. That means resource definition comes before attribute and permissions. Same in other places.

Reply:
👍

Reply:
Not addressed :(